source/renderer.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 const electron = require('electron');
+const {getCurrentWindow} = require('@electron/remote');
```

Comment:
See the `@electron/remote` readme.

Reply:
I'm not completely sure what you mean by that. If you mean check issues with getCurrentWindow, there are no event callbacks being installed, just the id of the window is being consumed. Since the `electron.remote.getCurrentWindow` was used to get the current window, I thought this would be the way to get it with @electron/remote since it replaces `electron.remote`.

Reply:
> Second, @electron/remote/main must be initialized in the main process before it can be used from the renderer: - https://github.com/electron/remote

Reply:
My thinking was the user would do that in their application before they use this module. IMO it's more transparent and that would eliminate the possibility of some side effects if @electron/remote/main is initialized multiple times (afaik there are none based on the code for initialization but it would be good to avoid that). If you think it would be better to initialize it for users, sure, I'll just add it to the readme.

Reply:
We can just initialize it here internally as the user is most likely going to use this in both the main and renderer process.